modules/swagger-codegen/src/main/resources/tizen/api-header.mustache

```diff
@@ -1,37 +1,74 @@
-#ifndef {{classname}}_H_
-#define {{classname}}_H_
+/*
+ * Copyright (c) 2016 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
@akhilkedia is it a must to include this particular line? For template files contributed by other companies or individuals, none of those includes the copyright note.